Annotations.md

```diff
@@ -1 +1,22 @@
-TODO
+# **Annotations**
+
+* Inline annotation of corpus data is a primary aim of Cosycat and is achieved through the use of a standard of marking within the hits retrieved from the corpus by a query that is stored in the general metadata, attached to the hit.
+
+* The user can use a combination of the query and annotation panes and the procedures described below to annotate data with a variety of tagset or create their own suitable for a particular case study.
+
+ ## **Annotation Process**
+ 
+ * Annotations are conducted in the annotation pane on hits marked in the query pane. The pane is then accessed through the query interface with the pane switch button.
+ 
+ * By opening the hit (either through clicking it or by opening all hits on the page with the open page button) a user can highlight the element to be annotated and, using the syntax [key]=[value] and pressing enter/return, attach an annotation to it to be saved to it. These annotations are displayed below the hit with separate keys on separate lines. Annotations cannot overlap on the same key line but can apply to the same item across keys. 
+ 
+  * Annotations can be edited or even deleted by clicking on the annotation and editing or deleting the value followed by enter/return. 
+  
+  * Single items (words or punctuation marks) can be easily selected by clicking the annotation entry box below the word.
+  
+  * Spans (multiple items in a hit) can also be annotated by selecting the annotation entry box and clicking and dragging across the span you’d like to annotate with one [key]=[value] combination (picture). This selection is persistent and remains after the annotation has been entered and can be deselected, with the entry pane returning to separating between single items, by double clicking the entry interface.
+  
+  * *Annotating across hits (Can we do this yet?)*
```

Comment:
do you mean creating a single annotation that spans several hits? that's not possible